modules/update_lease_status.tf

```diff
@@ -49,6 +54,19 @@ module "update_lease_status_lambda" {
   }
 }
 
+// Upload budget notification email templates to S3
```

Comment:
We do have some tfvars where we explicitly ask the user to provide s3 buckets and keys, and expect the user to upload the file themselves (eg. `var.reset_nuke_template_bucket` / `var.reset_nuke_template_key`). 

But changing the tfvars to accept s3 buckets/keys would be a breaking change, so I'd prefer to avoid it.